scripts/module/stadium.nut

```diff
@@ -0,0 +1,25 @@
+// Stadium billboards
+
+currentModelOffset <- 0;
+BillboardsBegin <- 2042;
+
+function onScriptLoad() {
+  billboard <- CreateObject(BillboardsBegin, 1, -1037.08, 1340.258, 36.552, 255);
+}
+
+function onPlayerCommand( player, cmd, text ) {
+  if (! player.Admin ) {
+    return;
+  }
+  cmd = cmd.tolower();
+  if (cmd == "sbb" || cmd == "swapbillboard") {
```

Comment:
not sure if we need the shortcut. More interesting would be to specify the ID [0-2, and possibly one to remove it] in my opinion.